pkg/vm/vm_ops_maths.go

```diff
@@ -94,6 +94,38 @@ func Dec(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
 	return NONE, nil
 }
 
+// Abs converts the Stack Item's value to an absolute value.
```

Comment:
Abs pops an integer off of the stack and pushes it's absolute value onto the stack.
Returns an error if the popped value is not an integer or if the absolute value cannot be taken

Reply:
:+1: